src/Groups/GAPGroups.jl

```diff
@@ -473,8 +471,9 @@ false, nothing
 ```
 """
 function isconjugate(G::GAPGroup, x::GAPGroupElem, y::GAPGroupElem)
-   if GAP.Globals.IsConjugate(G.X, x.X, y.X)
-      return true, group_element(G,GAP.Globals.RepresentativeAction(G.X, x.X, y.X))
+   conj = GAP.Globals.RepresentativeAction(G.X, x.X, y.X)
+   if conj != GAP.Globals.fail
+      return true, group_element(G, conj)
```

Comment:
Defining `isconjugate` such that two arguments are returned is probably inspired by the analogous design for Magma's `IsConjugate` function.

GAP's `IsConjugate` returns only the the `true`/`false` value.
If one is interested in a conjugating element, `RepresentativeAction` is the function to be called, where the object `fail` is returned if the two points are not in one orbit.

I think it is in general a good concept if one can choose in advance whether only the `true`/`false` is needed (the non-constructive variant) or also a conjugating element (the constructive variant).
Thus having two functions, like in GAP, would make  sense.

And in situations where one is interested in a conjugating element, it should be allowed to call the constructive variant also if no conjugating element exists, because otherwise one may have to do essentially the same computation twice.

Reply:
It seems Thomas and me are in full agreement here, great. We can address this later, I filed issue #120 as a reminder.